cli/src/pcluster/config/cluster_config.py

```diff
@@ -967,6 +968,7 @@ def __init__(
         self.additional_packages = additional_packages
         self.tags = tags
         self.iam = iam
+        self.config_region = config_region
```

Comment:
I think we can avoid this extra attribute and just set the `self.__region` in row 962

Reply:
I removed this line since the region of config has been parsed and used in controller layer. And `config_region` is left as an unused argument.

Reply:
Even removing this line fails the unit tests in a tricky way. I have to restore the code before your review. We can talk more